samples/ambient-argo/application/loadgen.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: loadgen
+  labels:
+    app: loadgen
+    service: loadgen
+spec:
+  ports:
+  - port: 8089
+    name: http
+  selector:
+    app: loadgen
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: loadgen
+  labels:
+    app: loadgen
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: loadgen
+  template:
+    metadata:
+      labels:
+        app: loadgen
+    spec:
+      containers:
+      - name: loadgen
+        image: therealmitchconnors/bookinfo-load-test:latest
```

Comment:
nit: any more official source of this?

Reply:
No, but I think after kubecon I'd like to build one as a part of the bookinfo app.  I end up configuring my own load generator every time and it's getting cumbersome.